src/components/Header/MenuButton.js

```diff
@@ -0,0 +1,129 @@
+import React from "react";
+import PageButton from "./PageButton";
+import {
+  Button,
+  Box,
+  Menu,
+  MenuItem,
+  useMediaQuery,
+  useTheme,
+} from "@mui/material";
+
+const MenuButton = ({ pages, selectedPage, handlePageClick }) => {
+  const theme = useTheme();
+  const isXsToMd = useMediaQuery(theme.breakpoints.between("xs", "md"));
+  const [anchorEl, setAnchorEl] = React.useState(null);
+
+  const backgroundImage = "linear-gradient(to right, #00F5A0, #00D9F5)";
+
+  const handleClick = (event) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  const handleMenuItemClick = (page) => {
+    handlePageClick(page);
+    handleClose();
+  };
+
+  if (isXsToMd) {
+    return (
+      <Box
+        sx={{
+          display: "inline-flex",
+          padding: {
+            xs: "1px",
+            sm: "1px",
+          },
+          borderRadius: "100px",
+          backgroundImage,
+          alignItems: "flex-start",
+          justifyContent: "center",
+          height: {xs: "100%",sm: "100%"},
+        }}
+      >
+        <Button
+          aria-controls="simple-menu"
+          aria-haspopup="true"
+          onClick={handleClick}
+          sx={{
+            fontFamily: "Inter",
+            fontWeight: 700,
+            fontStyle: "normal",
+            textAlign: "center",
+            fontSize: {
+              xs: "12px",
+              sm: "14px",
+            },
+            lineHeight: {
+              xs: "140%",
+              sm: "140%",
+            },
+            letterSpacing: {
+              xs: "0.2px",
+              sm: "0.2px",
+            },
+            textTransform: "none",
+            color: "#FFFFFF",
+            position: "relative",
+            backgroundColor: "#181A20",
+            borderRadius: "100px",
+            alignItems: "center",
+            justifyContent: "center",
+            alignSelf: "center",
+            height: {xs: "100%",sm: "36px"},
+            display: "flex",
+            gap: { xs: "4px", sm: "8px" },
+            padding: {xs: "8px", sm: "6px 16px"},
+          }}
```

Comment:
You can create a style.js file to abstract this as follows: 

Styles.js file
```
const style = {
  buttonProps: {
    sx: {
         color: "#FFFFFF"
         .....
         }
},
};
```

You also have access to the theme object: 

`sx: (theme) => ({
   color: theme.palette.secondary.main
})`

Where you import it

```
import styles from "./style.js"

<Button {...styles.buttonProps}></Button>
```